### PR TITLE
Fault Injection for Service Fabric (SF) - Parts Unlimited (Intergen)

### PR DESCRIPTION
**Pull request by steven2000nz**

### CODE REVIEW

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    Register-ServiceFabricApplicationType -ApplicationPathInImageStore 'Apps\WebApp'
+
+**Step 14.** Now we want to create a new Service Fabric application.
+
+    New-ServiceFabricApplication -ApplicationName 'fabric:/WebApp' -ApplicationTypeName 'WebAppType' -ApplicationTypeVersion 1.0
+
+**Step 15.** Go back to the Service Fabric dashboard and check to see if the application has deployed successfully.
+
+![](<media/azure11.png>)
+
```
Comment (julienstroheker) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: Please also add and show how can we have access to the application "from the outside" using the `<yourClusterName>.<yourRegion>.cloudapp.azure.com` url

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+###Task 1: Run some tests against the cluster.
+
+Now let's cause some mayhem. Service Fabric provides a 'chaos' scenario that will causes faults across the entire Service Fabric cluster.
+
+*"The chaos test scenario generates faults across the entire Service Fabric cluster. The scenario compresses faults generally seen over months or years to a few hours. The combination of interleaved faults with a high fault rate finds corner cases that would otherwise be missed."*
+
+**Step 1.** Run the following commands in PowerShell.
+
+Firstly we want to connect to the cluster (note you may already be connected)
+    Connect-ServiceFabricCluster [your-server-cluster-address].[location].cloudapp.azure.com:19000
```
Comment (julienstroheker) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: Please make sure this PowerShell line appear as a command section in the Markdown

**Comment by jonocairns** [comment]
Comments have been completed :)

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+    ...
+
+    "runtimes": {
+        "win10-x64": {}
+    }
+
+    ...
+}
+```
+
```
Comment (OguzPastirmaci) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: Please add an example for Windows 10 showing how the final version of the file would look like.

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+**Step 5.** Right click on the PartsUnlimitedWebsite and select 'Publish'
+
+**Step 6.** Create a new profile called 'FileSystem' then select 'Next'
+
```
Comment (OguzPastirmaci) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: How do I create a new profile? Add instructions and/or screenshots please.

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md (hunk 3 above)
Comment (jonocairns) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: I'm not quite sure what the ask here is - the example just above includes the windows 10 version.

"runtimes": {
 +        "win10-x64": {}
 +    }

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md (hunk 4 above)
Comment (jonocairns) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: 5 and 6 are the steps required to create a new profile. Should I combine them in to one to make it easier to read?

Comment (OguzPastirmaci) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: Please do.

File: docs/HOL-Fault_Injection_for_Service_Fabric/README.md (hunk 3 above)
Comment (jonocairns) [inline docs/HOL-Fault_Injection_for_Service_Fabric/README.md]: do you mean something like this ->

 `"runtimes": {
        "win10-x64": {},
        "win8-x64": {}
}`

**Pull request merged**